### ISSUE TITLE
Running Pytest

### ISSUE THREAD

**Issue by jacobjj** [jakeypakey/ECE_229_project#2]
Please provide instructions on how to run tests. Currently, it is not clear how to run these tests.

**Comment by EugeneLi5237** [comment]
Hi, `tests` can be run by calling `pytest filename.py`. If pytest is not installed, please run `pip install -U pytest`. Sorry for the inconvenience.

**Comment by jacobjj** [comment]
Running pytest, I get the following errors: 

```
FAILED test_accidentDelay.py::test_functions - FileNotFoundError
FAILED test_rawDataVisualization.py::test_get_processedAccidentLocationFile_1 - assert False
FAILED test_rawDataVisualization.py::test_get_processedAccidentLocationFile_3 - assert False
FAILED test_rawDataVisualization.py::test_get_processedComparisonFile_1 - assert False
```

Why is that?

**Comment by jakeypakey** [comment]
Hi Jacob, I am posting a private message on piazza regarding this.

**Comment by jacobjj** [comment]
Ok, I'll look into it. What's the code coverage of your tests?

**Comment by jakeypakey** [comment]
Because most of our code relates to data cleaning and visualization, most of our testing has to do with type checking the input to the methods.
